### PR TITLE
Partially resolve #10. Add a section about multiplayer

### LINKED ISSUE #10: Learning Chess

### PR DESCRIPTION
**Pull request by Danar435**

### CODE REVIEW

File: src/problem-analysis/problem-analysis.tex
Comment (audio-engineer) [inline src/problem-analysis/problem-analysis.tex:1]: Missing final newline

**Review by audio-engineer — CHANGES_REQUESTED** [review]

**Comment by audio-engineer** [comment]
Missing issue references in commits. You can manually move the issues now but next time use autoreference

**Review by audio-engineer — APPROVED** [review]

**Comment by Danar435** [comment]
I was hoping that naming the issue in the PR would associate it, oh well. But it works in my favor, as this PR only partially fixed the issue, it'll be fully fixed when @MadsHeilmann adds his section

**Review by SebastianMygind — APPROVED** [review]
nice

**Pull request merged**